UPGRADING.rst

```diff
@@ -11,6 +11,77 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#2246 Add deployment incarnation counter
+============================================================
+
+This PR adds a new environemnt variable `$AZUL_DEPLOYMENT_INCARNATION`. Service
```

Comment:
Too much information. Focus on the actual steps. No need to explain why. That new variable does not need to mentioned here.